### DISCUSSION TITLE
Associate Errors with fields ?

### DISCUSSION THREAD

**Discussion by Tekpakma** [houseform/houseform#55] [Q&A] [answered]
Hi, so i was playing around with the library and my first impressions were great. But im curious, would there be a non "hacky" way to associate Form errors with field (names)? Thinking about Object like errors.

**Comment by crutchcorn** [comment, marked as answer]
Hmm... This makes a lot of sense to me. I think v2 of HouseForm should have `errors` be an array. Maybe we can add a secondary field called `errorMap` to sidestep a breaking change for now.

Let me make a GitHub issue for this so that you can track it

**Reply by crutchcorn** [reply]
Issue can be found and tracked here: https://github.com/crutchcorn/houseform/issues/58
